README.md

```diff
@@ -1,10 +1,47 @@
-Oneup Contao Sentry Bundle
-==========================
+Contao Sentry Bundle
+====================
 
-This bundle provides an easy integration of sentry.io for Contao 4.4.x and newer.  
+This Contao bundle provides an easy integration of [sentry.io](https://sentry.io/) for Contao 4.4.x and newer.
 
 [![Author](http://img.shields.io/badge/author-@1upgmbh-blue.svg?style=flat-square)](https://twitter.com/1upgmbh)
 [![Software License](http://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE)
 [![Total Downloads](http://img.shields.io/packagist/dt/oneup/contao-sentry-bundle.svg?style=flat-square)](https://packagist.org/packages/oneup/contao-sentry-bundle)
 
 --
+
+In the first place, this is an "wrapper extension" for the [`sentry/sentry-symfony` bundle][1]. Therefore, you need to
+configure this bundle as you would configure the `sentry/sentry-symfony` bundle: [Documentation][2]
+
+### User feedback
+
+On the other hand you might want to implement the [User feedback][3] feature of sentry. The user feedback is primarily
+useful to let the users know that you've gotten notified about the issue and to let users give the opportunity to add
+some comments.
+
+In order to integrate this feature, you have to alter the error page template. Place a copy of 
+`vendor/contao/core-bundle/src/Resources/views/Error/layout.html.twig` in the directory 
+`app/Resources/ContaoCoreBundle/views/Error/`.
+
+Modify the copied template and place the following snippet just before the closing `</body>` tag:
+```twig
+    {% set sentry_id = ''|sentry_last_event_id %}
+    <script src="https://cdn.ravenjs.com/3.23.1/raven.min.js"></script>
+    {% if sentry_id %}
+        <script>
+            Raven.showReportDialog({
+                eventId: '{{ sentry_id }}',
+                dsn: 'https://12345678@sentry.io/1'
```

Comment:
Is it somehow possible to get the `dsn` from the config?

Reply:
We're not in a PHP context here but in a twig template. That's why I added the twig filter `sentry_last_event_id` — to grasp some variable from the PHP runtime.

Do you want the twig filter (i.e. `sentry_last_event_id`) to return both the sentry ID and the DSN? (Could be an assoc. array).

Reply:
Yes, we definitely can do that. But I'm also okay with leaving it as it is because the integration of the user feedback needs to be done manually nonetheless. Putting in the dsn by hand is no big deal then ;-)

Reply:
I get your point. But I don't like to configure the same value twice or more, so if it's already in the config, why not use it. :)